app/dashboard/models.py

```diff
@@ -2400,6 +2430,136 @@ def is_staff(self):
         """
         return self.user.is_staff if self.user else False
 
+    def calc_activity_level(self):
+        """Determines the activity level of a user
+
+        Returns:
+            str: High, Low, Medium, or New
+
+        """
+        high_threshold = 15
+        med_threshold = 15
+        new_threshold_days = 7
+
+        if self.created_on > (timezone.now() - timezone.timedelta(days=new_threshold_days)):
+            return "New"
+
+        visits = self.actions.filter(action='Visit')
+        visits_last_month = visits.filter(created_on__gt=timezone.now() - timezone.timedelta(days=30)).count()
+
+        if visits_last_month > high_threshold:
+            return "High"
+        if visits_last_month > med_threshold:
```

Comment:
`med_threshold` is the same as `high_threshold`, does that make sense? also - should there be an upper bound as well? (`visits_last_month` > `med_threshold` and <= `high_threshold`?)

Reply:
yeah.... it is.  

alos i just made this up... you guys tell me what the thresholds should be pls. in particular, @frankchen07 curious your opionion

Reply:
I think it's fine where it is 15 stands to be a visit every other day, medium is 2 visit per week, and lower than that is low.